src/recon_buildblock/BinNormalisationFromGEHDF5.cxx

```diff
@@ -323,14 +322,14 @@ read_norm_data(const string& filename)
     m_input_hdf5_sptr->initialise_efficiency_factors();
 
     // Do the reading using a buffer.
-    unsigned int total_size = scanner_ptr->get_num_rings()*scanner_ptr->get_num_detectors_per_ring();
+    unsigned long total_size = (scanner_ptr->get_num_rings()-1)*(scanner_ptr->get_num_detectors_per_ring()-1);
```

Comment:
using `long` won't work actually. You'd have to cast first, but in any case the `Array`  constructor takes `signed int`

Reply:
Oops some debugging changes that I didn't revert, my bad.